report/sections/Specification/Specification.tex

```diff
@@ -1,75 +1,72 @@
-This chapter presents an overview of our system using a system specification containing the purpose and the scope of our system, the target users, the constraints of the project, the use cases, and the prioritization using a MoSCoW analysis.
+This chapter presents an overview of our system using a system specification containing the purpose and the scope of our system, the target users, the constraints of the project, the use cases, and the feature prioritization using a MoSCoW analysis.
 
 \section*{Purpose and Scope of the System}
 The purpose of the system is to provide a better approach to learning functional programming in Haskell using a software application.
-We aim to make a system where students can solve exercises. The system should be able to provide feedback on whether the user's solution compiles, and whether it fulfills the requirements of the exercise.
+We aim to make a system where students can solve exercises. The system should be able to provide feedback on whether the user's solution compiles, and whether it fulfills the requirements of the given exercise.
 In addition, the lecturer should be able to add exercises and write specifications for exercise solutions.
 
-To reduce the scope of the project, we choose to focus on creating a platform for the \textit{Programming Paradigms} course, while ensuring the system abstractions support other languages, that could be supported in the future.
-Currently 172 students attend the course, thus we do not expect many of the following use cases to be performed at the same time.
+To reduce the scope of the project, we choose to focus on creating a platform for the \textit{Programming Paradigms} course while ensuring that the system can support other languages in the future.
+Currently, 172 students attend the course.
+Consequently, we do not expect most of the following use cases to occur at the same time.
```

Comment:
Should 10 be deleted and 9 be closer to 11?

I feel like 10 is kind of an odd insertion, but that 9 and 11 fits well together.

Reply:
done